go/runtime/host/sgx/epid.go

```diff
@@ -52,17 +44,54 @@ func (ep *teeStateEPID) Update(ctx context.Context, sp *sgxProvisioner, conn pro
 	}
 	supportsAttestationV1 := (regParams.TEEFeatures != nil && regParams.TEEFeatures.SGX.PCS)
 
+	// TODO: Could consider implementing a strategy for more optimized endpoint selection,
+	// but (re-)attestation are not so frequent that it would matter much.
+	for i, iasClient := range sp.ias {
+		resp, err := ep.update(ctx, sp, conn, report, nonce, supportsAttestationV1, iasClient)
+		switch {
+		case err == nil:
+			return resp, nil
+		case i == len(sp.ias)-1:
+			return nil, err
+		default:
+			sp.logger.Warn("error obtaining attestation, trying next IAS server", "err", err)
+			select {
+			case <-time.After(50 * time.Millisecond):
```

Comment:
Why do we need to wait?

Reply:
Seemed reasonable to backoff at least a tiny bit before retrying the next server